src/main/java/core/basesyntax/service/impl/ReaderServiceImpl.java

```diff
@@ -0,0 +1,26 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.service.ReaderService;
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Scanner;
+
+public class ReaderServiceImpl implements ReaderService {
+    @Override
+    public List<String> readFile(String filePath) {
+        Scanner scanner;
+        try {
+            scanner = new Scanner(new File(filePath));
+        } catch (FileNotFoundException e) {
+            throw new RuntimeException(e);
+        }
+        ArrayList<String> list = new ArrayList<>();
```

Comment:
You should use the List interface to declare 'list' instead of the concrete implementation 'ArrayList'. This follows the principle of programming to interfaces rather than implementations.